src/main/java/com/clickhouse/kafka/connect/sink/db/ClickHouseWriter.java

```diff
@@ -523,7 +525,7 @@ public void doInsertJson(List<Record> records) throws IOException, ExecutionExce
         Record first = records.get(0);
         String topic = first.getTopic();
         LOGGER.info(String.format("Number of records to insert %d to table name %s", batchSize, topic));
-        Table table = this.mapping.get(Utils.escapeTopicName(topic));
+        Table table = this.mapping.get(Utils.getTableName(topic, csc.getTopicToTableMap()));
```

Comment:
What about `describeTable`? We need to get metadata from another table  https://github.com/ClickHouse/clickhouse-kafka-connect/blob/374227d099367c0ae8480a4fae6dd8f63d5f90b6/src/main/java/com/clickhouse/kafka/connect/sink/db/helper/ClickHouseHelperClient.java#L135